src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -46,6 +46,10 @@ public void Start()
 
         public abstract class MergedTransactionCommand
         {
+            //overriding this property and setting it to false
```

Comment:
It was virtual until I changed it, but didn't change the comment. 
In the next PR for this issue I will fix it.